src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  display: flex;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  height: 70px;
+  margin: 20px 0;
+  width: 100%;
+}
+
+.icon-form {
+  display: flex;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  height: 42px;
+  margin: 20px 0;
+  width: 100%;
+}
+
+.field {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  border-radius: 4px;
+  width: 100%;
+  height: 100%;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+  outline: none;
+}
+
+.search--small {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+  cursor: pointer;
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search::placeholder {
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search--small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
```

Comment:
here you repeated `.search:hover`
do your really need it?

Reply:
no need, thank you